packages/console-types/src/types/endpoints.ts

```diff
@@ -47,6 +47,32 @@ const routeBooleanValue = {
   },
 } as const
 
+const routeRequestsPerSecond = {
+  type: 'object',
+  default: { inherited: true },
+  if: { type: 'object', properties: { inherited: { type: 'boolean', const: false } } },
+  then: {
+    type: 'object',
+    properties: {
+      inherited: { type: 'boolean', const: false },
+      requestsPerSecond: {
+        type: 'number',
+        description: 'The number of seconds to wait before the request is rejected',
```

Comment:
is this the correct description?

Reply:
Absolutely not. Should be something like `number representing limit on the requests that can be made in one second` I could propose a new description on another pull request.